### PR TITLE
Create AudioStrategy to display audio media

### PR DESCRIPTION
**Pull request by itkg-ngilain**
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/47
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/127

### CODE REVIEW

File: MediaBundle/Tests/DisplayMedia/Strategies/AudioStrategyTest.php [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+class AudioStrategyTest extends AbstractStrategyTest
+{
+    /**
+     * Set up the test
+     */
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->templating = Phake::mock('Symfony\Bundle\FrameworkBundle\Templating\EngineInterface');
```
Comment (nicolasThal) [inline MediaBundle/Tests/DisplayMedia/Strategies/AudioStrategyTest.php]: templating is not declared in the class properties

**Comment by itkg-ngilain** [comment]
@nicolasThal : updated

File: MediaBundle/Tests/DisplayMedia/Strategies/AudioStrategyTest.php [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+        Phake::when($this->container)->get('templating')->thenReturn($this->templating);
+
+        $this->strategy = new AudioStrategy($this->requestStack, '');
```
Comment (nicolasThal) [inline MediaBundle/Tests/DisplayMedia/Strategies/AudioStrategyTest.php]: there is already a base value for the second argument

**Comment by nicolasThal** [comment]
:+1: with the little modification

**Pull request merged**